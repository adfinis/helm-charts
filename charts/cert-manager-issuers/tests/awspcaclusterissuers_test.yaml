suite: AWSPCAClusterIssuer tests
templates:
  - awspcaclusterissuers.yaml
tests:
  - it: should create a AWSPCAIssuer
    values:
      - ../examples/awspcaissuer.yaml
    asserts:
      - matchSnapshot: {}
