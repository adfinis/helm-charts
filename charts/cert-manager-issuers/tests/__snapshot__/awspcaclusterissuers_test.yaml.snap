should create a AWSPCAIssuer:
  1: |
    apiVersion: awspca.cert-manager.io/v1beta1
    kind: AWSPCAClusterIssuer
    metadata:
      labels:
        app.kubernetes.io/app: cert-manager-issuers
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/version: "1"
        helm.sh/chart: cert-manager-issuers-0.3.0
      name: example
    spec:
      arn: <some-pca-arn>
